modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/main/java/com/liferay/dynamic/data/mapping/data/provider/internal/DDMDataProviderInvokeCommand.java

```diff
@@ -43,6 +43,9 @@ public DDMDataProviderInvokeCommand(
 				HystrixCommandProperties.Setter().
 					withExecutionTimeoutInMilliseconds(
 						_getTimeout(ddmRESTDataProviderSettings))
+			).andCommandPropertiesDefaults(
+			   HystrixCommandProperties.Setter()
+					.withExecutionIsolationSemaphoreMaxConcurrentRequests(20)
```

Comment:
@rijema How did you choose this number 20?

Reply:
According to the documentation, the Default Value is 10. https://github.com/Netflix/Hystrix/wiki/Configuration#execution.isolation.semaphore.maxConcurrentRequests

Reply:
@rijema I saw it, but why did you change it to 20? Did you perform some test to define this number?